src/humancellatlas/data/metadata/helpers/dss.py

```diff
@@ -92,14 +92,13 @@ def download_file(item):
     return bundle['version'], manifest, dict(metadata_files)
 
 
-def dss_client(deployment: Optional[str] = None) -> DSSClient:
+def dss_client(deployment: Optional[str] = 'prod') -> DSSClient:
     """
     Return a DSS client to DSS production or the specified DSS deployment.
 
-    :param deployment: The name of a DSS deployment like `dev`, `integration` or `staging`. If None, the production
-                       deployment (`prod`) will be used.
+    :param deployment: The name of a DSS deployment like `dev`, `integration` or `staging`.
     """
-    deployment = deployment + "." if deployment else ""
+    deployment = deployment + "." if deployment and deployment != 'prod' else ""
```

Comment:
… and we shouldn't accept False-ish values.

```suggestion
    deployment = "" if deployment == 'prod' else deployment + "." 
```

Also like to avoid negation as much as possible.

Reply:
Easier to read this way, agreed.